gap_analysis/models/business.py

```diff
@@ -0,0 +1,212 @@
+# -*- coding: utf-8 -*-
+# © <YEAR(2015)>
+# <AUTHOR(Elico Corp, contributor: Eric Caudal, Alex Duan, Xie XiaoPeng)>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+from openerp.osv import osv
+
+
+class BusinessEstimationLine(models.Model):
+    _name = "business.estimation.line"
+    _description = "Bus. Estimation Lines"
+
+    sequence = fields.Integer('Sequence')
+    name = fields.Char('Name', required=True)
+    business_requirement_id = fields.Many2one(
+        comodel_name='business.requirement',
+        string='Business Analysis',
+        ondelete='cascade'
+    )
+    type_id = fields.Many2one(
+        comodel_name='business.estimation.type',
+        string='Estimation Type',
+        ondelete='restrict'
+    )
+    estimated_time = fields.Integer(string='Estimated Time', required=True)
+    user_id = fields.Many2one(
+        comodel_name='res.users',
+        string='Assign To',
+        ondelete='set null'
+    )
+
+
+class BusinessRequirement(models.Model):
+    _name = "business.requirement"
+    _description = "Business Analysis"
+
+    @api.model
+    def _get_sequence(self):
+        sequence = self.env['ir.sequence'].get('business.requirement')
+        return sequence
+
+    sequence = fields.Char(
+        'Sequence',
+        readonly=True,
+        copy=False,
+    )
+    name = fields.Char(
+        'Name', required=True,
+        default=_get_sequence,
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    description = fields.Char(
+        'Description', required=True,
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    business_requirement = fields.Text(
+        'Customer Story',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    odoo_scenario = fields.Text(
+        'Odoo Scenario',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    odoo_gap = fields.Text(
```

Comment:
I dislike the `odoo` specific field names. Can you use just `scenario` and `gap`?